docs/reference/copy-guide.md

```diff
@@ -0,0 +1,275 @@
+# Gutencopy Guidelines
+
+## Longer Text
+Guidelines for writing multi-line/step instructions or narrative introductions/orientation to pages or features.
+
+This will obviously vary quite a lot depending on the context, but here are some general tips:
+
+#### ONE: Contractions are your friends!
+They’re more conversational, and a simple way to make text sound friendlier and less formal. (And they save a bit of space as well: a win-win.)
+
+#### TWO: Cut phrases that inflate your word count without actually adding meaning. 
+This happens frequently in two specific instances. First, when writing in the passive voice:
+
+> This block can be used to display single images.
+
+Any time you see phrases like “can be” or “is used”: halt. You’re writing in the passive voice. Try going active for a snappier (and shorter) sentence:
```

Comment:
Instead of "halt" (which admittedly did stop me reading the sentence), you could use a verb like "rephrase" or "reevaluate". That way you're telling them what to do and what to expect rather than just shutting them down (and getting the reader to pause without action for a moment, as I did.)

Reply:
I think saying halt is ok as got you to do what it was intended.